tests/unit/helper-functions/convert-token.spec.js

```diff
@@ -0,0 +1,14 @@
+import { convertToken } from '../../../src/popup/utils/helper';
+
+const invalidInput = [null, 'test', undefined, {}, [], NaN, '1', '-1'];
+
+describe('convertToken', () => {
+  [2, -2, 0].forEach((precision) => it('should move decimal point n places to desired direction', () => {
+    const value = 1234.9876;
+    const expectedResult = value * (10 ** precision);
```

Comment:
You've missed the point during creation of this test. We need to test that it converts with the maximum precision each digit should still be there. 
We need to test here edge cases something like 20 30 digits powering. Need to use BigInt or BigNumber here to test properly.